README.md

```diff
@@ -1,2 +1,45 @@
-# karma
+# Karma
```

Comment:
Whether "Karma" should be capitalised given that the repository name is not ... a difficult question.